frontend/components/Task/Create.vue

```diff
@@ -23,53 +23,33 @@ tasks:             全てのタスクのデータ
 -->
 <template>
   <div class="task-info">
```

Comment:
僕が忘れていたんですが、class名の変更お願いします…
```suggestion
  <div class="task-create">
```
それに伴い、styleタグ内のclass名も変更お願いします。

Reply:
修正しました。